extensions/amp-social-share/0.1/amp-social-share.js

```diff
@@ -85,10 +90,16 @@ class AmpSocialShare extends AMP.BaseElement {
   /** @private */
   handleClick_() {
     user().assert(this.href_ && this.target_, 'Clicked before href is set.');
-    const windowFeatures = 'resizable,scrollbars,width=640,height=480';
     const href = dev().assertString(this.href_);
     const target = dev().assertString(this.target_);
-    openWindowDialog(this.win, href, target, windowFeatures);
+    if (this.shareEndpoint_ === 'share:') {
+      dev().assert(navigator.share !== undefined,
+          'navigator.share disappeared.');
+      navigator['share'](parseQueryString(href.substr(href.indexOf('?'))));
```

Comment:
The `navigator['share']` (instead of `navigator.share`) formulation is needed to evade a test in gulp's `check-types` task, though presumably there's a better way… (This task runs on Travis; I can't see how to run it locally.)